aten/src/ATen/native/Distributions.cpp

```diff
@@ -179,6 +179,27 @@ Tensor& bernoulli_scalar_cpu_(Tensor& self, double p, Generator* gen) {
   return self;
 }
 
+DEFINE_DISPATCH(uniform_mkl_stub);
+
+Tensor& uniform_cpu_(Tensor& self, double from, double to, Generator* gen) {
+#if AT_MKL_ENABLED()
+  if (cpuinfo_initialize() && cpuinfo_vendor_intel == cpuinfo_get_processor(0)->core->vendor) {
+    uniform_mkl_stub(kCPU, self, from, to, gen);
+    return self;
+  }
+#endif
+  AT_DISPATCH_FLOATING_TYPES(self.scalar_type(), "uniform_cpu_", [&] {
+    CPUGenerator* generator = at::get_generator_or_default<CPUGenerator>(gen, detail::getDefaultCPUGenerator());
+    // See Note [Acquire lock when using random generators]
+    std::lock_guard<std::mutex> lock(generator->mutex_);
+    CPU_tensor_apply1<scalar_t>(
```

Comment:
Please use `TensorIterator` and `cpu_serial_kernel ` see https://github.com/pytorch/pytorch/pull/32534

Reply:
Done, `CPU_tensor_apply1` has been replaced by `TensorIterator`.